tests/derive_iden_tests.rs

```diff
@@ -0,0 +1,37 @@
+pub mod common;
+pub use common::{features::*, setup::*, TestContext};
+use sea_orm::entity::prelude::*;
+use sea_orm_macros::DeriveIden;
+
+#[derive(DeriveIden)]
+pub enum Class {
+    Id,
+    Title,
+    Text,
+}
+
+#[derive(DeriveIden)]
+struct Glyph;
+
+#[derive(DeriveIden)]
+pub enum Book {
+    Id,
+    #[sea_orm(iden = "turtle")]
+    Title,
+    #[sea_orm(iden = "TeXt")]
+    Text,
+}
+
+#[test]
+fn main() -> Result<(), DbErr> {
+    assert_eq!(Class::Id.to_string(), "id");
+    assert_eq!(Class::Title.to_string(), "title");
+    assert_eq!(Class::Text.to_string(), "text");
+
+    assert_eq!(Glyph.to_string(), "glyph");
+
+    assert_eq!(Book::Id.to_string(), "id");
+    assert_eq!(Book::Title.to_string(), "turtle");
+    assert_eq!(Book::Text.to_string(), "te_xt");
```

Comment:
I think this should be...?
```suggestion
    assert_eq!(Book::Text.to_string(), "TeXt");
```

Reply:
I think Iden in SeaQuery would make the name to be in snake case, no matter if it is from rename or the original name itself?
Having different behaviour from SeaQuery and SeaORM seems weird.

Reply:
oh wait 
maybe I was misreading the SeaQuery codes...
Seems like it doesn't change it after I re-read the code in SeaQuery.
Will change it back.

Reply:
Please correct me if I'm wrong. The original behaviour of `sea_query::Iden` should be...

```rs
#[derive(Iden)]
#[iden = "_glyphAbcD"]
struct Glyph;

assert_eq!(Glyph.to_string(), "_glyphAbcD");
```